commands/command_prune.go

```diff
@@ -86,6 +87,8 @@ func prune(fetchPruneConfig lfs.FetchPruneConfig, verifyRemote, dryRun, verbose
 	retainChan := make(chan string, 100)
 
 	gitscanner := lfs.NewGitScanner(nil)
+	gitscanner.Filter = filepathfilter.New(nil, cfg.FetchExcludePaths())
```

Comment:
Nice, I think we can avoid the `include` filters, since we only want to _not_ prune files that have been explicitly excluded from being downloaded in the first place.